api/apps/search/routes.js

```diff
@@ -0,0 +1,17 @@
+import search from '../../lib/elasticsearch.coffee'
```

Comment:
@kanaabe - you've now got [absolute URL ability](https://github.com/artsy/positron/blob/master/.babelrc#L12); I would use it! It makes refactoring, restructuring and readability way easier over time:

`import search from 'lib/elasticsearch.coffee'`

cc/ @eessex